### DISCUSSION TITLE
Can't access discord to ask about Touch of Zanzil Quest on Alliance

### DISCUSSION THREAD

**Discussion by SeraphTV** [chromiecraft/chromiecraft#3190] [General]
My discord name is SeraphTV#1950 I joined the discord to ask about the 2nd part of The Touch of Zanzil (A) Where you have to lay in front of Doc MixilPixil to "diagnose the issue" because no matter when or where you /lay down the quest doesn't complete. I read that this was reported on February 6th and am wondering how I'm supposed to progress when I can't stealth for an entire week.

**Comment by 55Honey** [comment]
Hello SeraphTV,
Thanks for reaching out.
Since the issue is already reported, we need to wait for a developer to fix it. It's not a game-breaking quest. One can just do it at a later stage if they aim for completion.
